micropython/files/mpconfigport.h

```diff
@@ -0,0 +1,16 @@
+/* This file is part of the MicroPython project, http://micropython.org/
```

Comment:
Not sure because it's technically "our own" config file but like 90% of it is copied from micropython...
and on a related note I'm not sure if we need to get include micropython's own license text anywhere now?

Reply:
It's probably best to leave it as-is.

Reply:
Also, since it's a standard MIT license, it's fine to just mark it as such in the port's Makefile, as you did.